Framework/PythonInterface/mantid/api/src/Exports/FileFinder.cpp

```diff
@@ -52,11 +63,17 @@ void export_FileFinder() {
                "Return a full path to the given file if it can be found within "
                "datasearch.directories paths. Directories can be ignored with "
                "ignoreDirs=True. An empty string is returned otherwise."))
-      .def("findRuns", &runFinderProxy, (arg("self"), arg("hintstr")),
+      .def("findRuns", &runFinderProxy,
```

Comment:
The Python API change is user-facing. I believe this should be noted in the release notes.

Reply:
@soininen I've updated `framework.rst`